React/Views/RCTScrollView.m

```diff
@@ -345,6 +345,12 @@ - (UIView *)hitTest:(CGPoint)point withEvent:(UIEvent *)event
   __block UIView *hitView;
 
   [_stickyHeaderIndices enumerateIndexesWithOptions:0 usingBlock:^(NSUInteger idx, BOOL *stop) {
+    NSUInteger subviewCount = [self contentView].reactSubviews.count;
+    if (idx >= subviewCount) {
+      RCTLogWarn(@"Sticky header index %zd was outside the range {0, %zd}", idx, subviewCount);
+      return;
```

Comment:
Seems to me that if this index is out of range then any later ones will be as well. Might as well halt the enumeration: `*stop = YES;`